src/stepper/advance.f90

```diff
@@ -216,6 +229,13 @@ subroutine adapt(bstep, t, dt)
             call diffy(svel(:, :, :, 3), ys)
             call fftxys2p(ys, dwdy)
 
+
+            ! Upper surface strain rms:
+            ! S = 0.5 * sqrt{(u_y + v_x)^2 + (u_x - v_y)^2}
+            ! v_x = u_y + zeta
+            ussrms = sum((two * dudy(nz, :, :) +  vor(nz, :, :, 3)) ** 2 + &
```

Comment:
I think you are missing the 0.5 multiplying sum; look at the comment 2 lines earlier

Reply:
I multiply later by 1/2 after getting the global sum. See https://github.com/matt-frey/ps3d/pull/71/files#diff-cef68695f14ebfde89f7eb2a68a4ea7636ec47f02124fe6c327b674589d1a0d7R318